lib/ui/map/location_detail_sheet.dart

```diff
@@ -248,40 +270,37 @@ class _LocationInformation extends StatelessWidget {
         Padding(
           padding: const EdgeInsets.all(8.0),
           child: Container(
-            height: 75,
```

Comment:
this part here improves the existing layout.
Old:
![old1](https://user-images.githubusercontent.com/10000439/79159481-6fc7e280-7dd8-11ea-81e6-e286ec12c85d.png)
![old2](https://user-images.githubusercontent.com/10000439/79159417-5161e700-7dd8-11ea-829c-87d5d908832a.png)

New:
![new1](https://user-images.githubusercontent.com/10000439/79159418-52931400-7dd8-11ea-852f-f349fda41826.png)
![new2](https://user-images.githubusercontent.com/10000439/79159421-52931400-7dd8-11ea-9320-ccd3a2e50d86.png)

Reply:
you have swapped old and new, right?

Reply:
No …